### ISSUE TITLE
[BUG] Media Info Combined View Season/Episode default focus position issue

### ISSUE THREAD

**Issue by ShibajiCh** [jurialmunkey/skin.arctic.horizon.2#635]
### Skin section

Library Views

### Current Behavior

Hi @jurialmunkey,

I noticed an issue with the Media Info Combined View for Seasons and Episodes. Suppose you browse the Seasons and Episodes in this view and for example stop the focus at Season 1 Episode 6 for a TV Show, then go back to another TV Show and browse it's Seasons and Episodes in the same view, then the focus will start from Season 1 Episode 6 where you left it at the last TV Show that you closed, instead of starting the focus at Season 1 Episode 1, or even better if it could focus on the next unwatched Episode for all TV Shows. Could you please check?

### Expected Behavior

The focus for the second opened TV Show should start at the beginning, i.e. Season 1 Episode 1, or even better if we could focus on the next unwatched Episode for all TV Shows.

### Steps To Reproduce

Go to any TV Show. Browse in Media Info Combined View. Focus on say Season 1 Episode 6. Go back. Open another TV Show. Browse in Media Info Combined View. The focus will start at the last viewed TV Show's position, i.e. Season 1 Episode 6.

### Checklist

- [X] I have made a separate issue for each bug that I am reporting.
- [X] I have searched the issues and this bug has not been reported.
- [X] I am using the most recent skin version.
- [X] I have confirmed that this bug does not occur in the default skin.

**Comment by jurialmunkey** [comment]
Yeah its really annoying. It's the same issue that happens in the hubs with the list selectors. 

Kodi controls which position in the list is focused after reloading the content and so far I haven't found a way to force the position back to one.

**Comment by ShibajiCh** [comment]
> Yeah its really annoying. It's the same issue that happens in the hubs with the list selectors.
> 
> Kodi controls which position in the list is focused after reloading the content and so far I haven't found a way to force the position back to one.

@jurialmunkey, oh okay fine. Hope you find a solution to this issue soon. Thank you so much!

Regards,
ShibajiCh.

**Comment by girkers** [comment]
I had never noticed this until I read this bug.

The reason I am commenting is that I was wanting it to go the next unwatched episode as well. I find it mildly annoying that you watch an episode and then it goes back to the very start of the list. i.e. watch episode 10 and when you are finished it goes back and displays episode 1.
